geomesa-convert/geomesa-convert-common/src/test/scala/org/locationtech/geomesa/convert/common/TransformersTest.scala

```diff
@@ -147,10 +147,27 @@ class TransformersTest extends Specification {
         "cast to int" >> {
           val exp = Transformers.parseTransform("$1::int")
           exp.eval(Array("", "1")) mustEqual 1
+          exp.eval(Array("", 1E2)) mustEqual 100
         }
         "cast to integer" >> {
           val exp = Transformers.parseTransform("$1::integer")
           exp.eval(Array("", "1")) mustEqual 1
+          exp.eval(Array("", 1E2)) mustEqual 100
```

Comment:
It actually works because double knows how to parse it. The part it doesn't work on is in the stringTo* functions.